_commands/basics/seq.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+seq
+-------
+### Overview
+
+`seq`prints a sequence of numbers to standard ouput.
+
+~~~ bash
+$ seq [options]... [*FIRST* [*STEP*]] *LAST*... 
+~~~
+
+<!--more-->
+
+where `seq` prints numbers from *FIRST* to *LAST* by an increment *STEP*.
+Default: *FIRST* & *STEP* are both 1. Each number is printed on its own line. Seq can be used on all real numbers.
+
+### Useful Options / Examples
+
+#### `-f, --format=`*FORMAT*`
+
+prints all numbers using *FORMAT*, where the default is `%g`. *FORMAT* must contain one of the following float output formats: `%g`, `%e`, `%f`. 
+
+EXAMPLE:
+
+~~~ bash
+$ seq -f "tyler%02" 10
+~~~
+
+will produce:
+
+tyler01
```

Comment:
Newlines don't show up, I think it's a Markdown thing. There's probably a way to force them, though.

![image](https://cloud.githubusercontent.com/assets/5882053/14057698/be1f6b26-f2de-11e5-848f-4633d18525ca.png)